pkg/query-service/app/server.go

```diff
@@ -52,7 +51,7 @@ func (s Server) HealthCheckStatus() chan healthcheck.Status {
 
 // NewServer creates and initializes Server
 // func NewServer(logger *zap.Logger, querySvc *querysvc.QueryService, options *QueryOptions, tracer opentracing.Tracer) (*Server, error) {
-func NewServer(serverOptions *ServerOptions) (*Server, error) {
+func NewServer(serverOptions *ServerOptions, storageURI string, datasource string) (*Server, error) {
```

Comment:
Is `storageURI` a URI or just storage type? If it is just storage type then shouldn't we name is as `storageType` or just `storage`?

Reply:
Good catch! I've renamed it to `storage`.